README.md

```diff
@@ -24,44 +24,33 @@ The React SDK is compatible with `React 16.3.0 +`
 import {
   createInstance,
   OptimizelyProvider,
-  OptimizelyExperiment,
-  OptimizelyVariation,
-  OptimizelyFeature,
+  useDecision,
 } from '@optimizely/react-sdk';
 
-const optimizely = createInstance({
+const optimizelyClient = createInstance({
   sdkKey: 'your-optimizely-sdk-key',
 });
 
+function MyComponent() {
+  const [decision] = useDecision('sort-algorithm');
+  return (
+    <React.Fragment>
+      <SearchComponent algorithm={decision.variables.algorithm} />
+      { decision.variationKey === 'bluebutton' && <BlueButton /> }

```

Comment:
I don't get how a blue vs green button relates to a 'sort-algorithm' flag.  Could variations be more like "relevant_first" and "popular_first" or "recommended_first" or something?